BrowserServicesKit/Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -580,20 +587,25 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
 #if os(macOS)
     private func loadAuthToken(from options: StartupOptions) async throws {
+        Logger.networkProtection.log("Load auth token")
         switch options.authToken {
         case .set(let newAuthToken):
+            Logger.networkProtection.log("Set new token: \(newAuthToken)")
```

Comment:
We shouldn't log the token.

Reply:
The value's log privacy default setting is `.private` the value is only visible in debug builds, but it's not useful anyway, I'll remove it.